charts/ccpay-bulkscanning-api/Chart.yaml

```diff
@@ -1,8 +1,12 @@
-apiVersion: v1
+apiVersion: v2
 appVersion: "1.0"
 description: A Helm chart for ccpay bulkscanning payments api
 name: ccpay-bulkscanning-api
 home: https://github.com/hmcts/ccpay-bulkscanning-app
-version: 1.0.6
+version: 1.0.8
 maintainers:
   - name: HMCTS Fees and Pay team
+dependencies:
+  - name: java
+    version: ~3.0.0-beta
```

Comment:
do not use a beta version when there's newer GA versions:
https://github.com/hmcts/chart-java/releases

Reply:
@timja  I have updated the chart version to 3.4.4 which is the latest one as per repo. Please confirm if it should be ok to use.